src/scripts/destinyTrackerApi/reviewDataCache.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'underscore';
+import { itemTransformer } from './itemTransformer.js';
+
+class reviewDataCache {
+  constructor() {
+    this._itemTransformer = new itemTransformer();
+    this._itemStores = [];
+  }
+
+  getMatchingItem(item) {
+    var dtrItem = this._itemTransformer.translateToDtrWeapon(item);
+
+    dtrItem.referenceId = String(dtrItem.referenceId);
```

Comment:
Shouldn't `translateToDtrWeapon` do that? (Also, why does it need to be a string?)